scripts/test/integration-dbt-async.sh

```diff
@@ -24,6 +32,17 @@ if [ "$DBT_VERSION" = "1.6" ] || [ "$DBT_VERSION" = "1.9" ]; then
     pip install "pydantic>2.11.0"
 fi
 
+# As on 28th April, 2025, the latest patch dbt-bigquery 1.6.13 on the 1.6 minor is not yet compatible with the latest
```

Comment:
I guess the fix https://github.com/dbt-labs/dbt-bigquery/pull/1061 needs to be back-ported to 1.6.x series. I have reported the same https://github.com/dbt-labs/dbt-adapters/issues/1031.

Meanwhile I am proposing to use `google-cloud-bigquery<3.31.0` for `dbt-bigquery 1.6.x`